vendor/knative.dev/operator/pkg/reconciler/knativeeventing/knativeeventing.go

```diff
@@ -104,6 +104,7 @@ func (r *Reconciler) ReconcileKind(ctx context.Context, ke *v1alpha1.KnativeEven
 		return err
 	}
 	stages := common.Stages{
+		common.FilterNamespace("knative-eventing"),
```

Comment:
So we exclude the namespace kind with the name.

Is patching vendored code really better, than manfiests?

Reply:
Yes, because manifests are in a versioned path directory

Reply:
which forces us to update patches every bump.